src/components/WebexLocalMedia/WebexLocalMedia.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {WebexAvatar} from '../';
+import {useMeeting} from '../hooks';
+
+import './WebexLocalMedia.scss';
+
+/**
+ * Webex Local Media component displays the user's local video
+ * previous to joining a meeting from a given meetingID.
+ *
+ * @param {object} props
+ * @returns {object} JSX of the component
+ */
+export default function WebexLocalMedia({meetingID, personID}) {
+  const {localVideo} = useMeeting(meetingID);
+  const media =
+    localVideo && Object.entries(localVideo).length > 0 ? (
+      <video controls>
```

Comment:
I don't think we should be including `controls`. Video should be controlled via meeting controls

Reply:
I added that to be able to control the video. I'm not sure how else to play the video, unless we don't need to.

Reply:
You can add "autoplay" but not sure how the controls would work with a `MediaStream`. We certainly don't want "browser controls" in a meeting experience